encryption_formats.md

```diff
@@ -150,8 +150,8 @@ Overhead: O = *padme_overhead(N + 1)* + 17b with
 
 #### Usage
 
-This is the default format used for simple resource encryption.
-It is safe for use in “one-time encryption” situation.
+This was the simple resource encryption format. It is no longer used.
```

Comment:
used to be?

Reply:
```suggestion
This used to be the simple resource encryption format. It is no longer used.
```

Reply:
Or use the `superseded` wording suggested above.

Reply:
Updated, per above